pages/index.js

```diff
@@ -0,0 +1,209 @@
+import * as Web3 from 'web3';
+import {
+    AppBar,
+    Box,
+    Button,
+    Chip,
+    Container,
+    Grid,
+    Input,
+    InputLabel,
+    Link,
+    MenuItem,
+    Select,
+    Toolbar,
+    Typography,
+} from '@mui/material';
+import { alpha, makeStyles } from '@material-ui/styles';
+import { useEffect, useState } from 'react';
+
+const useStyles = makeStyles((theme) => ({
```

Comment:
I believe this is the old approach to style with material ui. Since we are using the components from @mui/material do you want to refactor to use the new style system.

Refer https://next.material-ui.com/system/basics/ for more details.

If so we can uninstall all @material-ui/* in favor of @mui/*